common/hooks/pre-pkg/04-generate-runtime-deps.sh

```diff
@@ -34,15 +34,14 @@ add_rundep() {
 
 store_pkgdestdir_rundeps() {
         if [ -n "$run_depends" ]; then
-            : > ${PKGDESTDIR}/rdeps
             for f in ${run_depends}; do
                 _curdep="$(echo "$f" | sed -e 's,\(.*\)?.*,\1,')"
                 if [ -z "$($XBPS_UHELPER_CMD getpkgdepname ${_curdep} 2>/dev/null)" -a \
                      -z "$($XBPS_UHELPER_CMD getpkgname ${_curdep} 2>/dev/null)" ]; then
                     _curdep="${_curdep}>=0"
                 fi
-                printf -- "${_curdep} " >> ${PKGDESTDIR}/rdeps
-            done
+                printf -- "${_curdep}\n"
+            done | sort | xargs > ${PKGDESTDIR}/rdeps
```

Comment:
Can we just skip the `xargs`?

Reply:
the xargs brings it into one line, like it was before this. would it work if the rdeps/shlibs files are newline separated?

Reply:
the xargs brings it into a single line, like it was before. will it work if the files is newline-separated instead of space-separated?